term/fhir-term/src/test/resources/tx-expanded-valuesets/ValueSet-v2-0116.json

```diff
@@ -0,0 +1 @@
+{"resourceType" : "ValueSet","id" : "v2-0116","meta" : {"profile" : ["http://hl7.org/fhir/StructureDefinition/shareablevalueset"]},"language" : "en-US","url" : "http://terminology.hl7.org/ValueSet/v2-0116","version" : "2.9","name" : "v2.0116","title" : "v2 BED STATUS","status" : "active","experimental" : false,"immutable" : true,"expansion" : {"identifier" : "urn:uuid:a1719214-3697-4f7c-a248-fea62fda11d5","timestamp" : "2022-06-21T17:10:30.946Z","parameter" : [{"name" : "expansion-source","valueUri" : "ValueSet/v2-0116"},{"name" : "displayLanguage","valueString" : "en-US,en;q=0.9"},{"name" : "version","valueUri" : "http://terminology.hl7.org/CodeSystem/v2-0116|2.9"}],"contains" : [{"system" : "http://terminology.hl7.org/CodeSystem/v2-0116","code" : "C","display" : "Closed"},{"system" : "http://terminology.hl7.org/CodeSystem/v2-0116","code" : "H","display" : "Housekeeping"},{"system" : "http://terminology.hl7.org/CodeSystem/v2-0116","code" : "I","display" : "Isolated"},{"system" : "http://terminology.hl7.org/CodeSystem/v2-0116","code" : "K","display" : "Contaminated"},{"system" : "http://terminology.hl7.org/CodeSystem/v2-0116","code" : "O","display" : "Occupied"},{"system" : "http://terminology.hl7.org/CodeSystem/v2-0116","code" : "U","display" : "Unoccupied"}]}}
```

Comment:
do we want to pretty-print these? Not that it matters functionally, of course.

Reply:
sure.  done.